src/app/inbox/components/notification/notification.component.scss

```diff
@@ -23,7 +23,10 @@
 }
 
 .body-container {
-  padding: $spacing-base;
+  max-width: 800px;
```

Comment:
The margin/max-width do not seem to match the Zeplin sketch for in-app notifications